components/specification/released-schema/2016-DEV0/ome.xsd

```diff
@@ -3882,7 +3885,7 @@ PlaneInfo (as Plane)
       <xsd:complexContent>
         <xsd:extension base="Shape">
           <xsd:sequence>
-            <xsd:element ref="BinData" minOccurs="1" maxOccurs="unbounded"/>
+            <xsd:element ref="BinData" minOccurs="1" maxOccurs="1"/>
```

Comment:
@sbesson I'm not sure, I was just asking the question in case there was value in consistency here.  Leaving it is absolutely fine.